hack/swift/Makefile

```diff
@@ -84,6 +85,7 @@ byocni-up: rg-up net-up ## Brings up a swift cluster $name in $SUB/$REGION
 		--vnet-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/nodenet \
 		--pod-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/podnet \
 		--generate-ssh-keys \
+		--os-sku $(OSSKU)
```

Comment:
what happens if we don't specify this? jw because I thought that Ubuntu was the default.

Reply:
It does default to Ubuntu, just passing this parameter to setup the next P.R where I will create it with OS sku of windows. Thought it’s best to separate the P.R’s

Reply:
sounds good